src/Payum/Offline/Action/FillOrderDetailsAction.php

```diff
@@ -16,6 +16,10 @@ class FillOrderDetailsAction implements ActionInterface
     public function execute($request)
     {
         RequestNotSupportedException::assertSupports($this, $request);
+        
+        if($request->getToken()->getDetails()) {
+            return;
+        }
```

Comment:
@antonioperic 

`if($request->getToken()->getDetails()) {` - has to be removed, this class should not be aware of token and any such things,

we can set paid to false as default, Just mention it in the  UPGRADE